projects/ngx-quill/src/lib/quill-editor.component.ts

```diff
@@ -129,14 +111,14 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
   onValidatorChanged: () => void
 
   constructor(
-    @Inject(ElementRef) private elementRef: ElementRef,
-    @Inject(DomSanitizer) private domSanitizer: DomSanitizer,
-    @Inject(DOCUMENT) private doc: any,
-    @Inject(PLATFORM_ID) private platformId: any,
-    @Inject(Renderer2) private renderer: Renderer2,
-    @Inject(NgZone) private zone: NgZone,
+    public elementRef: ElementRef,
+    protected domSanitizer: DomSanitizer,
+    @Inject(DOCUMENT) protected doc: any,
+    @Inject(PLATFORM_ID) protected platformId: any,
+    protected renderer: Renderer2,
+    protected zone: NgZone,
     @Inject(QUILL_CONFIG_TOKEN) private config: QuillConfig,
-    @Inject(QuillService) private service: QuillService
+    protected service: QuillService
   ) {}
```

Comment:
`QuillEditorBase` constructor parameter accessibility modifiers have simply been changed from `private` to `protected` to allow for their access by derived classes / components.

`Inject` decorators have been moved back to `QuillEditorComponent` below.